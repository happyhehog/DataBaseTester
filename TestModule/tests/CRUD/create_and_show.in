[@Clear]
[@PrintLevel] MAIN

@@ ==============
@@ Testing CREATE
@@ ==============
[@Print] Testing CREATE\n

[@NoOutput]
CREATE TABLE t(id INT PRIMARY KEY NOT NULL, name CHAR(50));
[@NoOutput]
SHOW CREATE TABLE t;
[@NoOutput]
DROP TABLE t;
[@NoOutput]

[@NoOutput]
CREATE TABLE t(id INT PRIMARY KEY UNIQUE NOT NULL, name CHAR(50));
[@NoOutput]
SHOW CREATE TABLE t;
[@NoOutput]
DROP TABLE t;
[@NoOutput]

[@NoOutput]
CREATE TABLE t(id INT PRIMARY KEY NOT NULL UNIQUE, name CHAR(50));
[@NoOutput]
SHOW CREATE TABLE t;
[@NoOutput]
DROP TABLE t;
[@NoOutput]

[@NoOutput]
CREATE TABLE t(id INT UNIQUE, name CHAR(50));
[@NoOutput]
SHOW CREATE TABLE t;
[@NoOutput]
DROP TABLE t;
[@NoOutput]

[@NoOutput]
CREATE TABLE t(id INT UNIQUE, name CHAR(50) NOT NULL);
[@NoOutput]
SHOW CREATE TABLE t;
[@NoOutput]
DROP TABLE t;
[@NoOutput]

[@NoOutput]
CREATE TABLE t(id INT, name CHAR(50), age FLOAT, score INT);
[@NoOutput]
SHOW CREATE TABLE t;
[@NoOutput]
DROP TABLE t;
[@NoOutput]