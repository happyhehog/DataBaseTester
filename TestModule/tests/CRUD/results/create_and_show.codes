{"status":"OK","command":"SHOW CREATE TABLE t;","answer":"CREATE TABLE t(id INT PRIMARY KEY NOT NULL, name CHAR(50));"}
{"status":"OK","command":"SHOW CREATE TABLE t;","answer":"CREATE TABLE t(id INT PRIMARY KEY UNIQUE NOT NULL, name CHAR(50));"}
{"status":"OK","command":"SHOW CREATE TABLE t;","answer":"CREATE TABLE t(id INT PRIMARY KEY NOT NULL UNIQUE, name CHAR(50));"}
{"status":"OK","command":"SHOW CREATE TABLE t;","answer":"CREATE TABLE t(id INT UNIQUE, name CHAR(50));"}
{"status":"OK","command":"SHOW CREATE TABLE t;","answer":"CREATE TABLE t(id INT UNIQUE, name CHAR(50) NOT NULL);"}
{"status":"OK","command":"SHOW CREATE TABLE t;","answer":"CREATE TABLE t(id INT, name CHAR(50), age FLOAT, score INT);"}
