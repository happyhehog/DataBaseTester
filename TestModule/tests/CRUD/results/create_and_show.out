Testing CREATE

CREATE TABLE t(id INT PRIMARY KEY NOT NULL, name CHAR(50));
CREATE TABLE t(id INT PRIMARY KEY UNIQUE NOT NULL, name CHAR(50));
CREATE TABLE t(id INT PRIMARY KEY NOT NULL UNIQUE, name CHAR(50));
CREATE TABLE t(id INT UNIQUE, name CHAR(50));
CREATE TABLE t(id INT UNIQUE, name CHAR(50) NOT NULL);
CREATE TABLE t(id INT, name CHAR(50), age FLOAT, score INT);
